### PR TITLE
Updated path run_cmsearch to point to correct filtered rfam.cm file

### PR DESCRIPTION
**Pull request by ndliberial**
The path to Rfam.cm was updated to point to the ncrna subdirectory rather than the root assembly directory.
Code has tested on a new pipeline and it worked as needed.

### CODE REVIEW

**Review by thibauthourlier — CHANGES_REQUESTED** [review]
Hi Denye, could you please use `ncrna_dir` instead of `output_path`?
You will also need to update the extract_rfam_cm analysis to provide the path to the file and the script itself so it used the file provided

**Comment by ndliberial** [comment]
Hi Thibaut, 
Are you suggesting I change how the extract script is receiving the library name. In the current state, the last argument is taken as the file path

my $working_dir        = $ARGV[2]; #$self->param_required("working_dir");
my $output = path($working_dir . "/Rfam.cm");

**Comment by thibauthourlier** [comment]
Yes it should be something like `my $output = ARGV[2]` and working_dir is probably not needed anymore

**Comment by ndliberial** [comment]
Needed to change file. new PR was created.

**Comment by ndliberial** [comment]
requested changes effected now.

**Review by ndliberial — COMMENTED** [review]
Changes are in order

**Comment by ndliberial** [comment]
Changes now effected in both analysis

File: modules/Bio/EnsEMBL/Analysis/Hive/Config/Genome_annotation_conf.pm [hunk 1]
```diff
@@ -5146,7 +5146,7 @@ sub pipeline_analyses {
             cmd => "perl " . $self->o('sncrna_analysis_script') . "/filter_cm.pl " .
                              $self->o('rfam_cm') . ' ' .
                              $self->o('ncrna_dir') . '/accessions.txt ' .
-                             $self->o('ncrna_dir'),
+                             $self->o('ncrna_dir') . '/Rfam.cm',
```
Comment (thibauthourlier) [inline modules/Bio/EnsEMBL/Analysis/Hive/Config/Genome_annotation_conf.pm:5149]: Could you please use `filtered_rfam_cm`?

File: scripts/genebuild/sncrna/filter_cm.pl [hunk 2]
```diff
@@ -44,11 +44,11 @@ sub filter_rfam {
 
 my $rfam_cm_file       = $ARGV[0]; #$self->param_required('rfam_cm_file');
 my $rfam_accessions   = $ARGV[1]; #$self->param_required('rfam_accessions');
-my $working_dir        = $ARGV[2]; #$self->param_required("working_dir");
+#my $working_dir        = $ARGV[2]; #$self->param_required("working_dir");
```
Comment (thibauthourlier) [inline scripts/genebuild/sncrna/filter_cm.pl:47]: The line can be removed

**Review by thibauthourlier — CHANGES_REQUESTED** [review]

File: scripts/genebuild/sncrna/filter_cm.pl (hunk 2 above)
Comment (ndliberial) [reply scripts/genebuild/sncrna/filter_cm.pl]: Changes made as requested

**Review by thibauthourlier — APPROVED** [review]
Sounds good to me

**Comment by thibauthourlier** [comment]
I forgot to wait for Travis to be happy...

**Pull request merged**